config/cscs-ci.py

```diff
@@ -65,6 +65,24 @@ class ReframeSettings:
                     }
                 }
             },
+            'tsa': {
+                'descr': 'Tsa MCH',
+                'hostnames': [r'tsa-\w+\d+'],
```

Comment:
We used that expression to cover compute and post-processing nodes as well, since occasionally we need to start running the checks on specific nodes only for testing.